docs/src/content/docs/ops/backup-and-restore/overview.adoc

```diff
@@ -0,0 +1,54 @@
+---
+title: Backup and Restore
+---
+
+This document provides a high-level overview of XTDB’s stateful architecture and outlines what data is stored where, what typical backups include, and how to think about restoring XTDB in different failure scenarios.
+
+== Overview of Stateful Components
+
+XTDB consists of two primary stateful components:
+
+=== Log
+
+The log contains the record of all submitted transactions and inter-node messages.
+
+* The log is structured as a totally ordered, append-only sequence of entries.
+* Each transaction is recorded immutably and assigned a durable log offset.
+* The log guarantees ACID-compliant, serial execution of all writes across the cluster.
+
+From a backup perspective, the log is the source of truth for all changes made to the database. 
+Losing log entries that have not yet been indexed into storage risks permanent data loss.
+
+For more information on the available implementations of the transaction log, see the link:/ops/config/log[Log] docs.
+
+=== Storage Module
+
+The storage module contains the database’s state: tables, indexes, and supporting metadata.
+
+* Storage is structured as immutable, append-only files, ideal for backup and recovery.
+* Files represent snapshots of the database's state as derived from the transaction log.
+* Storage files can be treated as **immutable snapshots** tied to specific points in the database’s transactional history.
+
+For more information on the available implementations of the storage module, see the link:/ops/config/storage[Storage Module] docs.
+
+== What Data Is at Risk?
+
+[cols="1,3", options="header"]
+|===
+| Component | Risk if Lost
+
+| Log
+| All transactions on the log since the last indexed offset would be lost.
+
+| Storage Module
+| Without a backup or infinite retention on the log, all indexed data would be lost.
+
+| XTDB compute nodes
+| All in-memory state would be lost, but the contents of the log and storage module remain intact.
```

Comment:
Let's go a bit more reassuring here - something along the lines that the compute nodes only contain transient/cache data which is replicated durably elsewhere, and so compute nodes can come and go without data loss?